.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "name": "Mock app",
+      "port": 3000,
+      "request": "attach",
+      "type": "chrome",
+      "webRoot": "${workspaceFolder}"
+    }
+  ]
+}
```

Comment:
Should we add the `.vscode` folder and its contents (sub folders, files) to the `.gitignore` file to prevent unnecessary code from being pushed into the repository?